examples/darwin-framework-tool/commands/common/RemoteDataModelLogger.mm

```diff
@@ -191,5 +192,17 @@ CHIP_ERROR LogCommandErrorAsJSON(NSNumber * endpointId, NSNumber * clusterId, NS
     return LogError(value, status);
 }
 
+CHIP_ERROR LogGetCommissionerNodeId(NSNumber * value)
+{
+    VerifyOrReturnError(gDelegate != nullptr, CHIP_NO_ERROR);
+
+    Json::Value rootValue;
+    rootValue[kValueKey] = Json::Value();
+    rootValue[kValueKey][kNodeIdKey] = [value unsignedLongLongValue];
```

Comment:
Does this end up working correctly if the value is > 2^53?

Reply:
My understanding is that we ends up calling https://github.com/open-source-parsers/jsoncpp/blob/master/src/lib_json/json_value.cpp#L390 so I would say yes ?

Reply:
Right, but when we actually convert it to a JSON string, what happens?  Is it serialized as a number or string in the json?

Reply:
Per my understanding this is serialised as a number. That should not be an issue since we are not using JavaScript to read the value anymore is that is the problem you are thinking of ?

The following YAML test works fine for example: 
```
name: Test Cluster Tests                                                                                                                                                              
                                                                                                                                                                                      
config:                                                                                                                                                                               
    nodeId: 0x12344321                                                                                                                                                                
    cluster: "Unit Testing"                                                                                                                                                           
    endpoint: 1                                                                                                                                                                       
                                                                                                                                                                                      
tests:                                                                                                                                                                                
    - label: "Wait for the commissioned device to be retrieved"                                                                                                                       
      cluster: "DelayCommands"                                                                                                                                                        
      command: "WaitForCommissionee"                                                                                                                                                  
      arguments:                                                                                                                                                                      
          values:                                                                                                                                                                     
              - name: "nodeId"                                                                                                                                                        
                value: nodeId                                                                                                                                                         
                                                                                                                                                                                      
    # Tests for UInt64 attribute                                                                                                                                                      
                                                                                                                                                                                      
    - label: "Read attribute INT64U Default Value"                                                                                                                                    
      command: "readAttribute"                                                                                                                                                        
      attribute: "int64u"                                                                                                                                                             
      response:                                                                                                                                                                       
          value: 0                                                                                                                                                                    
                                                                                                                                                                                      
    - label: "Write attribute INT64U Max Value"                                                                                                                                       
      command: "writeAttribute"                                                                                                                                                       
      attribute: "int64u"                                                                                                                                                             
      arguments:                                                                                                                                                                      
          value: 18446744073709551615                                                                                                                                                 
                                                                                                                                                                                      
    - label: "Read attribute INT64U Max Value"                                                                                                                                        
      command: "readAttribute"                                                                                                                                                        
      attribute: "int64u"                                                                                                                                                             
      response:                                                                                                                                                                       
          value: 18446744073709551615                                                                                                                                                 
                                                                                                                                                                                      
    - label: "Write attribute INT64U Min Value"                                                                                                                                       
      command: "writeAttribute"                                                                                                                                                       
      attribute: "int64u"                                                                                                                                                             
      arguments:                                                                                                                                                                      
          value: 0                                                                                                                                                                    
                                                                                                                                                                                      
    - label: "Read attribute INT64U Min Value"                                                                                                                                        
      command: "readAttribute"                                                                                                                                                        
      attribute: "int64u"                                                                                                                                                             
      response:                                                                                                                                                                       
          value: 0
```

Reply:
Ah, as long as the parser can read numbers > 2^53 without dataloss, we are good.